README.md

```diff
@@ -2171,3 +2171,9 @@ My name is Kateryna and I'm happy to join this community!
 
 My name is [Dima](https://github.com/dimonlakhin). I decided to switch to a new profession. So I am glad to join the community. I hope to get your help and support.
 ![This is my own cat Shaya](./assets/images/dimon89.jpg)
+
+## Hi, everyone!
+My name is [Andrii Shchur](https://github.com/Acerpo). 
```

Comment:
I see empty spaces at the end of several of your lines. Please remove them as it is a bad habit. You can set your text editor (VSCode for example) to remove them automatically the moment you save your file.

Reply:
I do not understand what you're saying. How do you see empty spaces?

Installed Prettier, but I don't understand what has changed.

Reply:
Hi, I'm very sorry, must've caught a bug or something, I don't see any now, this was on my side. All fine with your line endings :)